A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -95,11 +95,11 @@ class CDUInitPage {
                     }
                 );
 
-                cruiseFlTemp = "_____|____[color]amber";
+                cruiseFlTemp = "_____\xa0|____[color]amber";
```

Comment:
This is also missing the degree symbol:

```suggestion
                cruiseFlTemp = "_____\xa0|___°[color]amber";
```

FMS 1A reference, but unlikely this changed:

![image](https://user-images.githubusercontent.com/929769/111835754-a0bc6b80-88f5-11eb-8660-6d295d650afa.png)